app/uk/gov/hmrc/claimvatenrolmentfrontend/forms/CaptureBox5FigureForm.scala

```diff
@@ -51,6 +51,11 @@ object CaptureBox5FigureForm {
   )
 
   val form: Form[String] = Form(
-      box5Figure -> text.verifying(box5FigureEmpty andThen box5FigureLength andThen box5FigureFormat)
```

Comment:
Please edit subject as DL-12557.
Changes looks good to me, As per the GDPR, the error should be above text field or below text field?

Reply:
If unit test is applicable, please add.